examples/plot_2_multidim_sigs.py

```diff
@@ -0,0 +1,249 @@
+"""
+2. 2D and 3D Signals
+====================
+Bycycle supports computing features for 2D and 3D arrays of signals, using
+:func:`~.compute_features_2d` or :func:`~.compute_features_3d`. These signals may be organized in a
+variety of ways, such as (n_epochs, n_timepoints), (n_channels, n_timepoints), or
+(n_channels,  n_epochs, n_timepoints). The ``axis`` keyword argument may be used to handle various
+organizations of signals. Understanding the axis argument is important to ensure features are
+computed in an appropriate manor.
+
+For 2D arrays:
+
+- Use ``axis = 0`` (default) when each signal is independent from others. This may be the case for
+  an (n_channels, n_timepoints) organization.
+- Use ``axis = None`` when each signal is dependent on others. This may be the case for an
+  (n_epochs, n_timepoints) organization, where each epoch was recorded continuously in a single
+  recording.
+- A 1D list of feature dataframes is returned.
+
+For 3d arrays:
+
+- Use ``axis = 0`` (default) when computing features independently across the first axis.
+  This is likely the case for a (n_channels, n_epochs, n_timepoints) organization.
+- Use ``axis = 1`` when computing features independently across the second axis. A possible
+  organization is (n_epochs, n_channels, n_timepoints).
+- Use ``axis = (0, 1)`` when computing features independently, for each signal, across both axes.
+  This may be the case for an (n_subjects, n_channels, n_timpoints) oraganization
+- Use ``axis = None`` when each signal is dependent on others. This may be the case for an
+  (n_runs, n_epochs, n_timpoints) oraganization, assuming a continuous recording.
+- A 2D list of feature dataframes is returned.
+
+"""
+
+####################################################################################################
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from neurodsp.sim import sim_combined
+from bycycle.group import compute_features_2d, compute_features_3d
+from bycycle.plts import plot_feature_categorical, plot_feature_hist
+
+####################################################################################################
+
+# Simulation settings
+n_seconds = 10
+fs = 500
+
+freq = 10
+f_range = (5, 15)
+
+epochs = 10
+channels = 5
+
+####################################################################################################
+
+####################################################################################################
+#
+# Example 1. 2D Array (n_epochs, n_timepoints)
```

Comment:
before you introduce this specific 2D example, introduce 2D examples in general here, and add the axis info (for 2D only) about here